src/pages/graphql/announcing-our-new-graphql-bootcamp.md

```diff
@@ -39,6 +39,6 @@ By the end of the week, you’ll have all you need to build Production-Ready Gra
 
 ## Show your interest in GraphQL training today
 
-We’re really excited about our new GraphQL Bootcamp and can’t wait to bring it to developers throughout the EU. We’re going to be announcing dates very soon so be sure you don’t miss out! If you’d like to learn how to harness the logical power of GraphQL, you can find out more about our GraphQL course [here](https://reactgraphql.academy/graphql/training/bootcamp/?utm_medium=direct&utm_source=blog&utm_campaign=graphql_exp&utm_content=graphql_announcement).
+We’re really excited about our new GraphQL Bootcamp and can’t wait to bring it to developers throughout the EU. We’re going to be announcing dates very soon so be sure you don’t miss out! If you’d like to learn how to harness the logical power of GraphQL, you can find out more about our GraphQL course [here](https://reactgraphql.academy/graphql/training/bootcamp/).
```

Comment:
@ric9176 internal links should not have the https and domain, otherwise, the Link component will treat them as a different site and won't do the HMTL5 history.pushState. Therefore a round trip to the server to load the new page